awesomplete.js

```diff
@@ -301,9 +309,7 @@ function $$(expr, con) {
 	return slice.call((con || document).querySelectorAll(expr));
 }
 
-$.create = function(tag, o) {
-	var element = document.createElement(tag);
-
+$.set = function(element, o) {
```

Comment:
> I did release the helpers: https://github.com/leaverou/bliss
Does it mean it can be added to Awesomplete dependencies? So we just use bliss and remove everything it provides from Awesomplete?